### PR TITLE
Adding the AMR, OCCA and RAJA versions of Laghos as sub-directories

### PR DESCRIPTION
**Pull request by tzanio**

### CODE REVIEW

File: raja/timing/timings_ray_2d_p1_ms10.org [hunk 1]
```diff
@@ -0,0 +1,65 @@
+|H1order|refs|H1dofs|L2dofs|H1cg_rate|L2cg_rate|forces_rate|update_quad_rate|total_time|total_rate|
```
Comment (vladotomov) [inline raja/timing/timings_ray_2d_p1_ms10.org:1]: Do we need to add these?

File: raja/cub/eclipse code style profile.xml [hunk 2]
```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (vladotomov) [inline raja/cub/eclipse code style profile.xml:1]: What is this?

File: raja/cub/.settings/org.eclipse.cdt.codan.core.prefs [hunk 3]
```diff
@@ -0,0 +1,72 @@
+eclipse.preferences.version=1
```
Comment (vladotomov) [inline raja/cub/.settings/org.eclipse.cdt.codan.core.prefs:1]: Did this come from the eclipse editor?

File: raja/cub/.cproject [hunk 4]
```diff
@@ -0,0 +1,1223 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (vladotomov) [inline raja/cub/.cproject:1]: Eclipse?

File: raja/timing/timings_ray_2d_p1_ms10.org (hunk 1 above)
Comment (tzanio) [reply raja/timing/timings_ray_2d_p1_ms10.org:1]: I believe @camierjs is planning to clean this directory. For now I just added it as it was in the branch.

**Comment by tzanio** [comment]
@jakubcerveny, do you mind verifying the amr sub-directory?

**Comment by tzanio** [comment]
Alternative to this PR, we can perform the same changes in each of the branches and merge them one-by-one. This will have the added benefit of attributing the commits appropriately :)

**Comment by jakubcerveny** [comment]
The problem with the AMR version is that it's a little hacky (LagrangianHydroOperator::AMRUpdate) and needs a slightly modified version of MFEM (laghos-amr-dev, the thing that's needed is BlockVector::Update that always updates the block vector), we would have to merge that first.

**Comment by tzanio** [comment]
> The problem with the AMR version is that it's a little hacky (LagrangianHydroOperator::AMRUpdate) 

I think that is fine for miniapp purposes

>  needs a slightly modified version of MFEM (laghos-amr-dev, the thing that's needed is BlockVector::Update that always updates the block vector)

Lets do this. Can you propose a minimal PR?

**Comment by tzanio** [comment]
Deprecated in favor of #26, #27, etc.

**Pull request closed without merging**